single-node-k8s.sh

```diff
@@ -110,6 +110,7 @@ function deploy_rook()
 {
     ${GOPATH}/src/github.com/ceph/ceph-csi/scripts/minikube.sh deploy-rook
     ${GOPATH}/src/github.com/ceph/ceph-csi/scripts/minikube.sh create-block-pool
+    ${GOPATH}/src/github.com/ceph/ceph-csi/scripts/minikube.sh create-ec-pool
```

Comment:
`create-ec-pool`  to `create-block-ec-pool` we also have filesystem EC pools lets make the option specific to block

Reply:
Okay :+1: